auction-server/src/api.rs

```diff
@@ -301,6 +312,7 @@ pub async fn start_api(run_options: RunOptions, store: Arc<Store>) -> Result<()>
     #[openapi(
     paths(
     bid::bid,
+    bid::solana_bid,
```

Comment:
Let me think on how we can make it backward compatible with evm sdk for now.

Reply:
I think it would be a little bit tricky to merge both apis for now. lets separate the solana endpoint and we can have them merged after we have more idea around the solana part and we can make better decisions around api design.

Reply:
endpoints are merged now :D